### PR TITLE
ACS-6112: re-enable disabled test

### PR DESCRIPTION
**Pull request by krdabrowski**
Re-enable testSecondaryAncestorWithNodeHavingComplexSecondaryRelationship

- open debug port for live-indexer in E2Es

### CODE REVIEW

**Review by azakrzewski-hy — APPROVED** [review]

File: tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java [hunk 1]
```diff
@@ -194,16 +194,18 @@ private void startOrFail(Startable... startables)
...
-                       .withEnv("ALFRESCO_ACCEPTEDCONTENTMEDIATYPESCACHE_BASEURL", "http://transform-core-aio:8090/transform/config");
+        return new GenericContainer<>(getImagesConfig().getLiveIndexingImage())
+            .withNetwork(network)
+            .withNetworkAliases("live-indexing")
+            .withEnv("ELASTICSEARCH_INDEXNAME", CUSTOM_ALFRESCO_INDEX)
+            .withEnv("SPRING_ELASTICSEARCH_REST_URIS", "http://elasticsearch:9200")
+            .withEnv("SPRING_ACTIVEMQ_BROKERURL", "nio://activemq:61616")
+            .withEnv("ALFRESCO_SHAREDFILESTORE_BASEURL", "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file/")
+            .withEnv("ALFRESCO_ACCEPTEDCONTENTMEDIATYPESCACHE_BASEURL", "http://transform-core-aio:8090/transform/config")
+            .withEnv("JAVA_TOOL_OPTIONS", "-agentlib:jdwp=transport=dt_socket,address=*:5005,server=y,suspend=n")
```
Comment (mpichura) [inline tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java:207]: 5005 because ACS is opening 8000, correct?

**Review by mpichura — APPROVED** [review]
LGTM

File: tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java (hunk 1 above)
Comment (krdabrowski) [reply tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java:207]: Yes, and also this port is often used for live-indexer debugging... or at least I think so after seeing our d-c files 😉  ps. 5006 is often used for re-indexer debugging

Comment (mpichura) [reply tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java:207]: Thanks. BTW - do not merge this PR before GA release is finished.

**Pull request merged**